optimizely/optimizely.py

```diff
@@ -322,28 +322,18 @@ def track(self, event_key, user_id, attributes=None, event_tags=None):
       self.logger.info('Not tracking user "%s" for event "%s".' % (user_id, event_key))
       return
 
-    # Filter out experiments that are not running or that do not include the user in audience
-    # conditions and then determine the decision i.e. the corresponding variation
-    decisions = self._get_decisions(event, user_id, attributes)
```

Comment:
`_get_decisions` can be deleted now as it's dead code, right?

Reply:
Good catch. Updating.